File: app/src/main/java/knf/kuma/tv/cards/RecordCardView.java
package knf.kuma.tv.cards;

import android.content.Context;
import android.widget.ImageView;
import android.widget.TextView;

import butterknife.BindView;
import butterknife.ButterKnife;
import knf.kuma.R;
import knf.kuma.commons.PatternUtil;
import knf.kuma.commons.PicassoSingle;
import knf.kuma.pojos.RecordObject;
import knf.kuma.tv.BindableCardView;

public class RecordCardView extends BindableCardView<RecordObject> {

    @BindView(R.id.img)
    ImageView imageView;
    @BindView(R.id.title)
    TextView title;
    @BindView(R.id.chapter)
    TextView chapter;

    public RecordCardView(Context context) {
        super(context);
        ButterKnife.bind(this);
    }

    @Override
    public void bind(RecordObject data) {
        PicassoSingle.get(context).load(PatternUtil.getCover(data.aid)).into(imageView);
        title.setText(data.name);
        chapter.setText(data.chapter);
    }

    @Override
    public ImageView getImageView() {
        return imageView;
    }

    @Override
    public int getLayoutResource() {
        return R.layout.item_tv_card_chapter;
    }
}

File: app/src/main/java/knf/kuma/tv/cards/RecordCardView.kt
package knf.kuma.tv.cards

import android.content.Context
import android.widget.ImageView
import android.widget.TextView
import knf.kuma.R
import knf.kuma.commons.loadGlide
import knf.kuma.pojos.av1.Record
import knf.kuma.tv.BindableCardView
import org.jetbrains.anko.find

class RecordCardView(context: Context) : BindableCardView<Record>(context) {

    override val imageView: ImageView
        get() = find(R.id.img)
    override val layoutResource: Int
        get() = R.layout.item_tv_card_chapter

    override fun bind(data: Record) {
        imageView.loadGlide(data.imageUrl())
        find<TextView>(R.id.title).text = data.name
        find<TextView>(R.id.chapter).text = data.chapter()
    }
}
